pa/private.go

```
@@ -0,0 +1,420 @@
+		n.High = rewriteExpr(n.High, offset)
+		n.Max = rewriteExpr(n.Max, offset)
+		n.X = rewriteExpr(n.X, offset)
+		return dumpExpr(n, n.Lbrack-offset)
+	case *ast.IndexExpr:
+		n.Index = rewriteExpr(n.Index, offset)
+		n.X = rewriteExpr(n.X, offset)
+		return dumpExpr(n, n.Lbrack-offset)
+	case *ast.UnaryExpr:
+		n.X = rewriteExpr(n.X, offset)
+		if n.Op == token.AND {
+			// TODO: actually invalid, see &s == &s example, which fails due to taking address of copies of s
```

golangci-lint

pa/private.go:184: Line contains TODO/BUG/FIXME: "TODO: actually invalid, see &s == &s exa..."

(godox)